wiki/Game_mode/osu!/id.md

```diff
@@ -0,0 +1,291 @@
+  - Tidak mengumpulkan slider tick, yang terlihat ataupun tidak, tidak akan membuat Anda ketinggalan, tetapi akan **mengatur ulang** kombo menjadi **0**.
+  - Gagal menyelesaikan slider sampai akhir tidak akan menyebabkan miss tetapi **tidak** akan menambah kombo.
+- Sebuah miss diberikan jika pemain tidak mengumpulkan slider tick.
+
+**Spinner:**
+
+- Skor 300, 100, atau 50 diberikan tergantung pada jumlah putaran yang dilakukan dibandingkan dengan panjang spinner.
+- Miss diberikan jika pemain tidak dapat memperluas lingkaran spinner ke batas yang telah disesuaikan untuk mendapatkan skor 50
+  - Pada skin v1 : Miss diberikan jika pemain tidak dapat memenuhi meteran yang ada di kedua sisi spinner hingga batas yang telah disesuaikan untuk mendapatkan skor 50.
+
+### Skor/Pengganda Kombo
+
+**berikut adalah hal-hal yang akan menambahkan satu poin ke combo multiplier:**
+
+- Ketukan yang tepat pada hit circle.
+- Ketukan yang berhasil di awal slider.
+- Bola slider yang berhasil mengumpulkan slider tick.
+- Bola slider yang berhasil sampai pada ujung slider atau reverse arrow pada slider.
+- Spinner yang selesai/setengah selesai
+
+**Berikut ini akan mengatur ulang combo multiplier menjadi nol:**
+
+- Miss pada hit circle.
+- Menekan slider terlalu awal.
+- Miss pada awal slider atau slider tick
+- Gagal memperluas lingkaran spinner ke ukuran yang telah disesuaikan.
+  - Pada skin v1 : Gagal mengisi setidaknya setengah dari meteran spinner.
+
+**Hal berikut tidak akan menambah atau mengatur ulang combo multiplier:**
+
+- Miss pada ujung slider (akan menghasilkan Katu atau 100/50 hit)
+- Spinner dengan bonus poin 1,000
+
+## Bilah HP
+
+bilah HP adalah sistem yang digunakan untung menghitung *health drain* dan *health gain*, penjelasannya cukup rumit sehingga tidak akan dijelaskan secara detail. Semuanya disimpan di pengaturan kesulitan HP yang hanya dapat diatur oleh pembuat map itu sendiri
+
+**Berikut ini hal yang dapat memulihkan HP:**
+
+- Ketukan yang berhasil pada hit circle (penilaian yang lebih baik menghasilkan lebih banyak HP; hit circle terakhir dalam kombo akan memberikan lebih banyak HP).
+- Ketukan yang berhasil pada awal slider.
+- Bola slider yang berhasil mengumpulkan slider tick.
+- Bola slider yang berhasil menyentuh ujung atau reverse arrow dari slider (penilaian yang lebih baik di titik akhir akan menghasilkan lebih banyak HP).
+- Menyelesaikan spinner dan tetap memutar spinner selama durasi spinner (penilaian yang lebih baik di akhir spinner menghasilkan lebih banyak perolehan HP).
+
+**Berikut ini adalah hal-hal yang akan menguras HP**
+
+- Pengurasan HP konstan/*health drain*
+- Sebuah miss.
+- Miss pada hit circle.
+- Menekan slider terlalu awal.
+- Miss pada slider tick atau slider reverse arrow.
+- Gagal dalam menyelesaikan sebuah spinner untuk mendapatkan setidaknya nilai 50.
+
+**Berikut ini adalah hal yang akan menahan HP pada bilah HP**
```

Comment:
all three items here seem to be missing the colon (`:`) as follows:

`**Berikut ini adalah hal-hal yang dapat memulihkan nilai HP pada health bar:**`
`**Berikut ini adalah hal-hal yang dapat menguras nilai HPp ada health bar:**`
`**Berikut ini adalah hal yang tidak memengaruhi nilai HP pada health bar:**`